sdk/textanalytics/Azure.AI.TextAnalytics.Authoring/README.md

```diff
@@ -0,0 +1,90 @@
+# Azure Authoring client library for .NET
+
+This section should give out brief introduction of the client library.
```

Comment:
The README needs to be updated at some point. Do you have a separate tracking issue?